CHANGELOG.md

```diff
@@ -87,6 +87,8 @@
 - Integrating latest versions of GeoZones and GeoLogos for territories.
   Especially using history of towns, counties and regions from GeoHisto.
   [#499](https://github.com/opendatateam/udata/issues/499)
+- Placeholders are now entirely provided by themes
```

Comment:
Maybe explicit that we're talking about *images* placeholders, I was confused, vs input fields placeholders.